File: Day43/FirstAPI/Models/Product.cs
namespace FirstAPI.Models
{
    public class Product :IEquatable<Product>
    {
        public int Id { get; set; }
        public string? Name { get; set; }
        public float Price { get; set; }
        public int Quantity { get; set; }

        public bool Equals(Product? other)
        {
            return this.Id == other.Id;
        }
    }
}

File: Day71/myapp/src/components/Product.js
import React from "react";
import './Product.css';



function Product(props){
    // var Prod={
    //     "name":"Pencil",
    //     "price":5,
    //     "quantity":10
    // };


    return(
        <div className="alert alert-primary Product">
            <h2>Product Details</h2>
            Product Name : {props.p.name}
            <br/>
            Product Price : ${props.p.price}
            <br/>
            Product Quantity : {props.p.quantity}
            <br/>
        </div>
    );
}

export default Product;